docs/docs/migrating-from-v0-to-v1.md

```diff
@@ -24,24 +24,24 @@ git mv utils src
 
 ## Replace react-router's Link component with gatsby-link
 
-Add new package to project.
+`gatsby-link` is a wrapper for the `<Link>` component in react-router.  It automatically prefixes urls and handles prefetching.  Add `gatsby-link` to your project by running:
```

Comment:
Because automatically prefixing is quite different than in 0.x — make that more empasized. Perhaps keep this line but also restore the "Prefixing links is also now handled automatically by our new `<Link>` component so remove usages of `prefixLink` in links."